{
  "uid" : "4cafe1029fcecaea",
  "name" : "test_returns_true",
  "fullName" : "tests.brownie.token.tests.test_transferFrom#test_returns_true",
  "historyId" : "592db1f5b525cbb8a4e298823c44ebef",
  "time" : {
    "start" : 1639074286145,
    "stop" : 1639074286145,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "ValueError: Gas estimation failed: 'Transaction simulation failed: Error processing Instruction 1: invalid account data for instruction'. This transaction will likely revert. If you wish to broadcast, you must set the gas limit manually.",
  "statusTrace" : "brownie_project = <Project 'TokenProject'>\naccounts = <brownie.network.account.Accounts object at 0x7f17c09d9100>\n\n    @pytest.fixture(scope=\"module\")\n    def token(brownie_project, accounts):\n>       return brownie_project.Token.deploy(\"Test Token\", \"TST\", 18, 1e21, {'from': accounts[0]})\n\ntests/brownie/token/tests/conftest.py:49: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n../../../.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/network/contract.py:600: in __call__\n    return tx[\"from\"].deploy(\n../../../.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/network/account.py:508: in deploy\n    receipt, exc = self._make_transaction(\n../../../.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/network/account.py:725: in _make_transaction\n    raise VirtualMachineError(e) from None\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = VirtualMachineError(ValueError(\"Gas estimation failed: 'Transaction simulation failed: Error processing Instruction 1:...or instruction'. This transaction will likely revert. If you wish to broadcast, you must set the gas limit manually.\"))\nexc = \"Gas estimation failed: 'Transaction simulation failed: Error processing Instruction 1: invalid account data for instruction'. This transaction will likely revert. If you wish to broadcast, you must set the gas limit manually.\"\n\n    def __init__(self, exc: ValueError) -> None:\n        try:\n            exc = exc.args[0]\n        except Exception:\n            pass\n    \n        if isinstance(exc, dict) and \"message\" in exc:\n            if \"data\" not in exc:\n                raise ValueError(exc[\"message\"]) from None\n    \n            self.message: str = exc[\"message\"].rstrip(\".\")\n    \n            if isinstance(exc[\"data\"], str):\n                # handle parity exceptions - this logic probably is not perfect\n                if \"0x08c379a0\" in exc[\"data\"]:\n                    revert_type, err_msg = [i.strip() for i in exc[\"data\"].split(\"0x08c379a0\", 1)]\n                    err_msg = eth_abi.decode_abi([\"string\"], HexBytes(err_msg))\n                    err_msg = f\"{revert_type} '{err_msg}'\"\n                elif exc[\"data\"].endswith(\"0x\"):\n                    err_msg = exc[\"data\"][:-2].strip()\n                else:\n                    err_msg = exc[\"data\"]\n                raise ValueError(f\"{self.message}: {err_msg}\") from None\n    \n            try:\n                txid, data = next((k, v) for k, v in exc[\"data\"].items() if k.startswith(\"0x\"))\n            except StopIteration:\n                raise ValueError(exc[\"message\"]) from None\n    \n            self.txid: str = txid\n            self.source: str = \"\"\n            self.revert_type: str = data[\"error\"]\n            self.pc: Optional[str] = data.get(\"program_counter\")\n            if self.pc and self.revert_type == \"revert\":\n                self.pc -= 1\n    \n            self.revert_msg: Optional[str] = data.get(\"reason\")\n            self.dev_revert_msg = brownie.project.build._get_dev_revert(self.pc)\n            if self.revert_msg is None and self.revert_type in (\"revert\", \"invalid opcode\"):\n                self.revert_msg = self.dev_revert_msg\n            elif self.revert_msg == \"Failed assertion\":\n                self.revert_msg = self.dev_revert_msg or self.revert_msg\n    \n        else:\n>           raise ValueError(str(exc)) from None\nE           ValueError: Gas estimation failed: 'Transaction simulation failed: Error processing Instruction 1: invalid account data for instruction'. This transaction will likely revert. If you wish to broadcast, you must set the gas limit manually.\n\n../../../.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/exceptions.py:121: ValueError",
  "flaky" : false,
  "newFailed" : false,
  "newBroken" : false,
  "newPassed" : false,
  "retriesCount" : 0,
  "retriesStatusChange" : false,
  "beforeStages" : [ {
    "name" : "configure_neon_network",
    "time" : {
      "start" : 1639074263656,
      "stop" : 1639074263656,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "brownie_project",
    "time" : {
      "start" : 1639074265439,
      "stop" : 1639074280877,
      "duration" : 15438
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "accounts",
    "time" : {
      "start" : 1639074281690,
      "stop" : 1639074282034,
      "duration" : 344
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "setup_env",
    "time" : {
      "start" : 1639074281690,
      "stop" : 1639074281690,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "token",
    "time" : {
      "start" : 1639074285051,
      "stop" : 1639074285611,
      "duration" : 560
    },
    "status" : "broken",
    "statusMessage" : "ValueError: Gas estimation failed: 'Transaction simulation failed: Error processing Instruction 1: invalid account data for instruction'. This transaction will likely revert. If you wish to broadcast, you must set the gas limit manually.\n",
    "statusTrace" : "  File \"/home/runner/.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/_pytest/fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/_pytest/fixtures.py\", line 932, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/runner/work/neon-compatibility/neon-compatibility/tests/brownie/token/tests/conftest.py\", line 49, in token\n    return brownie_project.Token.deploy(\"Test Token\", \"TST\", 18, 1e21, {'from': accounts[0]})\n  File \"/home/runner/.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/network/contract.py\", line 600, in __call__\n    return tx[\"from\"].deploy(\n  File \"/home/runner/.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/network/account.py\", line 508, in deploy\n    receipt, exc = self._make_transaction(\n  File \"/home/runner/.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/network/account.py\", line 725, in _make_transaction\n    raise VirtualMachineError(e) from None\n  File \"/home/runner/.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/exceptions.py\", line 121, in __init__\n    raise ValueError(str(exc)) from None\n",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  } ],
  "afterStages" : [ {
    "name" : "accounts::0",
    "time" : {
      "start" : 1639074497661,
      "stop" : 1639074497661,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "feature",
    "value" : "Brownie"
  }, {
    "name" : "parentSuite",
    "value" : "tests.brownie.token.tests"
  }, {
    "name" : "suite",
    "value" : "test_transferFrom"
  }, {
    "name" : "host",
    "value" : "fv-az83-54"
  }, {
    "name" : "thread",
    "value" : "4662-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.brownie.token.tests.test_transferFrom"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 55,
        "skipped" : 0,
        "passed" : 0,
        "unknown" : 0,
        "total" : 55
      },
      "items" : [ {
        "uid" : "6e8b6a30111e5c5a",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/187//#testresult/6e8b6a30111e5c5a",
        "status" : "broken",
        "statusDetails" : "brownie.exceptions.RPCRequestError: 'EthereumModel' object has no attribute 'debug_traceTransaction'",
        "time" : {
          "start" : 1639053360570,
          "stop" : 1639053372828,
          "duration" : 12258
        }
      }, {
        "uid" : "c294c1c2e401f758",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/188//#testresult/c294c1c2e401f758",
        "status" : "broken",
        "statusDetails" : "brownie.exceptions.RPCRequestError: 'EthereumModel' object has no attribute 'debug_traceTransaction'",
        "time" : {
          "start" : 1639056823249,
          "stop" : 1639056827116,
          "duration" : 3867
        }
      }, {
        "uid" : "7e6a14af564fb182",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/186//#testresult/7e6a14af564fb182",
        "status" : "broken",
        "statusDetails" : "ValueError: Verifying nonce before send transaction: Error processing Instruction 1: invalid program argument",
        "time" : {
          "start" : 1639045901927,
          "stop" : 1639045901927,
          "duration" : 0
        }
      }, {
        "uid" : "45ae1da615e49100",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/184//#testresult/45ae1da615e49100",
        "status" : "broken",
        "statusDetails" : "ValueError: Verifying nonce before send transaction: Error processing Instruction 1: invalid program argument",
        "time" : {
          "start" : 1639019990212,
          "stop" : 1639019990212,
          "duration" : 0
        }
      }, {
        "uid" : "548dc251aa75d006",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/183//#testresult/548dc251aa75d006",
        "status" : "broken",
        "statusDetails" : "ValueError: Verifying nonce before send transaction: Error processing Instruction 1: invalid program argument",
        "time" : {
          "start" : 1639010480482,
          "stop" : 1639010511860,
          "duration" : 31378
        }
      }, {
        "uid" : "865267489580c129",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/182//#testresult/865267489580c129",
        "status" : "broken",
        "statusDetails" : "brownie.exceptions.RPCRequestError: 'EthereumModel' object has no attribute 'debug_traceTransaction'",
        "time" : {
          "start" : 1638990700223,
          "stop" : 1638990704862,
          "duration" : 4639
        }
      }, {
        "uid" : "5cf96f81a52d4930",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/177//#testresult/5cf96f81a52d4930",
        "status" : "broken",
        "statusDetails" : "brownie.exceptions.RPCRequestError: 'EthereumModel' object has no attribute 'debug_traceTransaction'",
        "time" : {
          "start" : 1638969961990,
          "stop" : 1638969976319,
          "duration" : 14329
        }
      }, {
        "uid" : "bae0d9298fa00c30",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/181//#testresult/bae0d9298fa00c30",
        "status" : "broken",
        "statusDetails" : "ValueError: Verifying nonce before send transaction: Error processing Instruction 1: invalid program argument",
        "time" : {
          "start" : 1638981046296,
          "stop" : 1638981046296,
          "duration" : 0
        }
      }, {
        "uid" : "7846df96aebc4a5f",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/180//#testresult/7846df96aebc4a5f",
        "status" : "broken",
        "statusDetails" : "ValueError: Verifying nonce before send transaction: Error processing Instruction 1: invalid program argument",
        "time" : {
          "start" : 1638975844689,
          "stop" : 1638975844689,
          "duration" : 0
        }
      }, {
        "uid" : "5192fec9b3b12cad",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/179//#testresult/5192fec9b3b12cad",
        "status" : "broken",
        "statusDetails" : "brownie.exceptions.RPCRequestError: 'EthereumModel' object has no attribute 'debug_traceTransaction'",
        "time" : {
          "start" : 1638974678533,
          "stop" : 1638974683785,
          "duration" : 5252
        }
      }, {
        "uid" : "da1d4eee826cdfc1",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/178//#testresult/da1d4eee826cdfc1",
        "status" : "broken",
        "statusDetails" : "brownie.exceptions.RPCRequestError: 'EthereumModel' object has no attribute 'debug_traceTransaction'",
        "time" : {
          "start" : 1638973594131,
          "stop" : 1638973637665,
          "duration" : 43534
        }
      }, {
        "uid" : "dfdf51c154b2439a",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/176//#testresult/dfdf51c154b2439a",
        "status" : "broken",
        "statusDetails" : "brownie.exceptions.RPCRequestError: 'EthereumModel' object has no attribute 'debug_traceTransaction'",
        "time" : {
          "start" : 1638951295561,
          "stop" : 1638951308174,
          "duration" : 12613
        }
      }, {
        "uid" : "4758466fc46e7285",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/174//#testresult/4758466fc46e7285",
        "status" : "broken",
        "statusDetails" : "requests.exceptions.ConnectionError: HTTPSConnectionPool(host='proxy.teststand2.neontest.xyz', port=443): Max retries exceeded with url: /solana (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fadbd81f0d0>: Failed to establish a new connection: [Errno 111] Connection refused'))",
        "time" : {
          "start" : 1638932977995,
          "stop" : 1638932977995,
          "duration" : 0
        }
      }, {
        "uid" : "7990658d086c4b60",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/173//#testresult/7990658d086c4b60",
        "status" : "broken",
        "statusDetails" : "requests.exceptions.ConnectionError: HTTPSConnectionPool(host='proxy.teststand2.neontest.xyz', port=443): Max retries exceeded with url: /solana (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fb865a3b130>: Failed to establish a new connection: [Errno 111] Connection refused'))",
        "time" : {
          "start" : 1638923386941,
          "stop" : 1638923386941,
          "duration" : 0
        }
      }, {
        "uid" : "c9f084d9a8718283",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/172//#testresult/c9f084d9a8718283",
        "status" : "broken",
        "statusDetails" : "brownie.exceptions.RPCRequestError: 'EthereumModel' object has no attribute 'debug_traceTransaction'",
        "time" : {
          "start" : 1638913331554,
          "stop" : 1638913343864,
          "duration" : 12310
        }
      }, {
        "uid" : "24d204eee2531d0",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/171//#testresult/24d204eee2531d0",
        "status" : "broken",
        "statusDetails" : "requests.exceptions.ConnectionError: HTTPSConnectionPool(host='proxy.testnet.neonlabs.org', port=443): Max retries exceeded with url: /solana (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f240840a520>: Failed to establish a new connection: [Errno 111] Connection refused'))",
        "time" : {
          "start" : 1638912579776,
          "stop" : 1638912579776,
          "duration" : 0
        }
      }, {
        "uid" : "8d26e82e9b620ca7",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/170//#testresult/8d26e82e9b620ca7",
        "status" : "broken",
        "statusDetails" : "brownie.exceptions.RPCRequestError: 'EthereumModel' object has no attribute 'debug_traceTransaction'",
        "time" : {
          "start" : 1638908718989,
          "stop" : 1638908732684,
          "duration" : 13695
        }
      }, {
        "uid" : "73b8e3d11d35c66f",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/169//#testresult/73b8e3d11d35c66f",
        "status" : "broken",
        "statusDetails" : "brownie.exceptions.RPCRequestError: 'EthereumModel' object has no attribute 'debug_traceTransaction'",
        "time" : {
          "start" : 1638892696410,
          "stop" : 1638892699736,
          "duration" : 3326
        }
      }, {
        "uid" : "62dc42fc38b3afc3",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/168//#testresult/62dc42fc38b3afc3",
        "status" : "broken",
        "statusDetails" : "ValueError: Gas estimation failed: 'Transaction simulation failed: Error processing Instruction 1: invalid account data for instruction'. This transaction will likely revert. If you wish to broadcast, you must set the gas limit manually.",
        "time" : {
          "start" : 1638891851795,
          "stop" : 1638891851795,
          "duration" : 0
        }
      }, {
        "uid" : "bdcce070151f645c",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/167//#testresult/bdcce070151f645c",
        "status" : "broken",
        "statusDetails" : "brownie.exceptions.RPCRequestError: 'EthereumModel' object has no attribute 'debug_traceTransaction'",
        "time" : {
          "start" : 1638863192642,
          "stop" : 1638863197316,
          "duration" : 4674
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "4cafe1029fcecaea.json",
  "parameterValues" : [ ]
}